docs/releases.rst

```diff
@@ -14,6 +14,10 @@ support for :ref:`CPU cores</spec/hardware/cpu>` HW requirement.
 support for :ref:`Virtualization hypervisor</spec/hardware/virtualization>`
 HW requirement.
 
+New section :ref:`review` describing benefits and various forms of
```

Comment:
I'm not sure this is worth of release note. It has no direct impact on features or future plans (e.g. spec changes)

Reply:
From one point of view I agree with you, but on the other hand the pull request reviews have a **super important impact** on all the features, as without them we are not moving anyway. So, I would like to take any opportunity to highlight them, remind them and emphasize how important they are and which benefits they bring :)

Reply:
Fair enough. My PoV is that 'release' is something I would read to know about new features or changes. Not about the development workflow which, IMO is not interesting to the User.

Reply:
Got it. Almost convinced to remove the section, but cannot resist the last question: Don't we want to encourage even users that they are invited and welcome to contribute to the project? Review being one of the possible ways how anybody, including users, can help the project to move on? ;-)

Reply:
I would leave it there :)

Reply:
Your call. But release notes aren't for development news IMO, thats all.

Reply:
Not sure about this one. I'm thinking, it is a notable change linked to a release. Is there another place for it?